qucs/qucs/octave_window.cpp

```diff
@@ -117,7 +117,7 @@ bool OctaveWindow::startOctave()
   octProcess.waitForStarted();
 
   if(octProcess.state()!=QProcess::Running) {
-    output->setText(tr("ERROR: Failed to execute " + Program));
+    output->setText(tr("ERROR: Failed to execute ") + Program);
     return false;
```

Comment:
certainly, you are right. i will/might fix it some day. don't hold your breath.

is this an imperfection or a bug?